packages/loader/container-loader/src/container.ts

```diff
@@ -1399,6 +1415,16 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private processRemoteMessage(message: ISequencedDocumentMessage): IProcessMessageResult {
         const local = this._clientId === message.clientId;
 
+        // Report if we're getting messages from a clientId older than the most recent
+        if (!local && this.clientIdHistory.includes(message.clientId)) {
+            this.logger.sendErrorEvent({
+                eventName: "matchedOldClientIdInRemoteMessage",
+                clientId: this._clientId,
+                messageClientId: message.clientId,
+                historyIndex: this.clientIdHistory.indexOf(message.clientId),
```

Comment:
i don't think historyIndex is that useful from the telemetry, as there isn't much we can do with it outside the process. i think we should log the sequence number, and the client sequence number.

Reply:
i figured it would give an idea of the recency of the message relative to the clientid changes, but i guess that can also be deduced from the rest of the telemetry

Reply:
that makes sense. fine to keep it, but let's add seq, and client seq